app/views/layouts/_item_form.html.erb

```diff
@@ -5,42 +5,30 @@
          <%= form_with model: @item, local: true do |f| %>
             <div class="panel panel-default panel-body">
                <div class="form-group">
-                  <label class="col-sm-2 control-label">Name</label>
```

Comment:
this didn't use rails form_helper standards so i needed to change them to follow naming conventions. i also tested in dev and was able to see the form and it worked correctly

Reply:
Good catch!